website/admin.py

```diff
@@ -1,3 +1,32 @@
 from django.contrib import admin
 
-# Register your models here.
+from website.models import Course, Macrotopic, Microtopic, VideoLesson, ExerciseList, UserProfile, User
+
+
+class CourseAdmin(admin.ModelAdmin):
+    fields = ('macrotopic', 'title')
```

Comment:
The model of Course object has been updated but the fields in here are not;
This causes an error when try to create a course on admin page because these fields doesnt have the property ```null=True``` and they are not shown in the page that creates a course, consequently, these fields dont get any value and they are null.

```suggestion
    fields = ('macrotopic', 'title', 'description')
    readonly_fields = ('ratings', 'posts')
```